frontend/src/metabase/dashboard/hooks/use-dashboard-url-query.ts

```diff
@@ -122,7 +122,7 @@ export function useDashboardUrlQuery(
   }, [router, location, selectedTab, dispatch]);
 }
 
-const QUERY_PARAMS_ALLOW_LIST = ["objectId"];
+const QUERY_PARAMS_ALLOW_LIST = ["objectId", "locale"];
```

Comment:
I wonder if we should also update the parameters allow list for the [`SyncedParametersList` component](https://github.com/metabase/metabase/blob/e441f1d58c312f807940cbb70c69cb0ab137a937/frontend/src/metabase/query_builder/components/SyncedParametersList.tsx#L78) (used for native question parameters)?

Reply:
💯 yes!
Thank you for pointing that out, let me change one of the e2e tests to cover that scenario

Reply:
Awesome! It'd be great to unify this logic at some point, but it's kind of hard to do while native question parameters are using `history` instead of `react-router-redux`. #44639 should have more context around that